src/fragments/lib/auth/ios/signout/10_local_signout.mdx

```diff
@@ -23,14 +34,33 @@ func signOutLocally() async {
 func signOutLocally() -> AnyCancellable {
     Amplify.Publisher.create {
         await Amplify.Auth.signOut()
-    }.sink {
-        if case let .failure(authError) = $0 {
-            print("Sign out failed with error \(authError)")
+    }.sink(
+        receiveCompletion: {
+            if case let .failure(authError) = $0 {
+                print("Sign out failed with error \(authError)")
+            }
```

Comment:
wondering this is still valid? when would we get this failure?

Reply:
Good catch. It appears that the `receiveCompletion` block here is unnecessary, but required because `Amplify.Publisher.create` returns `AnyPublisher<_, Error>` even when a non-throwing function is provided as its argument. I updated the code snippets in 6af0cef and made a [PR in Amplify](https://github.com/aws-amplify/amplify-ios/pull/2414) to add a non-throwing overload of `create(_:)`.